docs/source/roadmap.md

```diff
@@ -209,33 +135,126 @@ SeqFISH implements a decoder that carries out a local search of adjacent pixels
 - [ ] CC-by licensing
 - [ ] Documented upload procedures to s3
```

Comment:
Split this section into "data handling" and "developer focus point". Add a deliverable that covers the BioFormats tool to the data handling section. 

Serve Data + Licensing move to the "data handling section"

Reply:
Let's do this in your branch.